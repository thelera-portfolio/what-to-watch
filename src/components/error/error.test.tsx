import * as React from "react";
import * as renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import Error from "./error";

it(`Error is rendered correctly`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <Error
          message={`new error`}
        />
      </BrowserRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
