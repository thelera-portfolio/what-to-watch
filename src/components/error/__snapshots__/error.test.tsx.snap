// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error is rendered correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal-content"
    style={
      Object {
        "color": "red",
        "padding": "20px",
      }
    }
  >
    <p>
      new error
    </p>
  </div>
</div>
`;
